<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>patrol_main_window</class>
 <widget class="QMainWindow" name="patrol_main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Db">
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <addaction name="actConnect"/>
    <addaction name="actDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <addaction name="menu_Db"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actConnect">
   <property name="text">
    <string>&amp;Connect ...</string>
   </property>
  </action>
  <action name="actDisconnect">
   <property name="text">
    <string>&amp;Disconnect ...</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
